### ISSUE TITLE
HTML attribute example incorrectly fixed by Prettier

### ISSUE THREAD

**Issue by joyously** [mdn/content#21603]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes

### What specific section or headline is this issue about?

Boolean Attributes

### What information was incorrect, unhelpful, or incomplete?

The `<div itemscope="itemscope">` part is shown twice with one saying valid and one saying invalid.
```
<div itemscope>This is valid HTML but invalid XML.</div>
<div itemscope="itemscope">This is also valid HTML but invalid XML.</div>
<div itemscope="">This is valid HTML and also valid XML.</div>
<div itemscope="itemscope">
  This is also valid HTML and XML, but perhaps a bit verbose.
</div>
```

### What did you expect to see?

I think one of the `<div itemscope="itemscope">` should be `<div itemscope="true">` because of the sentence following the example, but it doesn't seem like it would be valid HTML or XML.
I don't know what it should be, but it shouldn't have the same thing labeled two ways.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/attributes`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/attributes/index.md
* Last commit: https://github.com/mdn/content/commit/4b2f9de5cf7a4eb7b684960b6b9f5f5c35693181
* Document last modified: 2022-09-19T18:29:03.000Z

</details>

**Comment by bsmth** [comment]
I've opened a PR which should fix this in https://github.com/mdn/content/pull/21683

**Comment by teoli2003** [comment]
Fixed by #21683
